### DISCUSSION TITLE
Custom position for manual test cases

### DISCUSSION THREAD

**Discussion by spartedo** [allure-framework/allure2#1716] [TestOps]
When dragging any manual test case in suite, you can drag it only to another suite:
![image](https://user-images.githubusercontent.com/29703462/182883876-7d3b2010-b9cb-4fa9-affd-f710f6ed2b81.png)
But you can't drag it to same suite, to custom/another position.

It will be useful to have custom position for cases and sorting variant like sorting by: id, Name, Status or Created date

**Comment by cheshi-mantu** [comment]
custom position is not possible as the representation of test cases in the test cases tree is dynamic.
the sortiong you are mentionin is available, you can apply soriting by clicking the arrows near id, name, status, created date.
Although the sorting by created date will be the same as the sortin by the id as the later is the created date the bigger is the id as the ID is defined in the DB and assigned as next available.
